### PR TITLE
docs: Updating Contributing Requirements

### PR DESCRIPTION
**Pull request by prateek3255**
## Summary of change
- Update Node version requirement
- Add a step in tests to add a static entry for localhost.org
- Remove the conflicting eslint and prettier files for TestingApp

## Related issues
N/A

## Test Plan
N/A

## Documentation changes
N/A

## Checklist for important updates
- [x] Changelog has been updated
- [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
   - Along with the associated array in `lib/ts/version.ts`
- [ ] Changes to the version if needed
   - In `package.json`
   - In `package-lock.json`
   - In `lib/ts/version.ts`
- [x] Had run `npm run build-pretty`
- [x] Had installed and ran the pre-commit hook
- [x] Issue this PR against the latest non released version branch.
   - To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
   - If no such branch exists, then create one from the latest released branch.

## Remaining TODOs for this PR
N/A

### CODE REVIEW

File: TestingApp/.prettierrc.js [hunk 1]
```diff
@@ -1,7 +0,0 @@
-module.exports = {
```
Comment (rishabhpoddar) [inline TestingApp/.prettierrc.js:1]: you removed this, however there aren't any changes to the actual formatting of the code in the TestingApp. Are you sure this is the right thing to do? And run prettier?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ We're so excited you're interested in helping with SuperTokens! We are happy to
 ## Development Setup
```
Comment (rishabhpoddar) [inline CONTRIBUTING.md:14]: an extension to this PR - can you add a PR check for github to run the pre commit hook like it's done in other repos?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: TestingApp/.prettierrc.js (hunk 1 above)
Comment (prateek3255) [reply TestingApp/.prettierrc.js:1]: The test files in `TestingApp` have always been formatted and linted against the root ESLint and prettier, but these files had different rules than the root ones, which caused the editor to show a lot of formatting errors:

image

I have also updated the prettier commands in `package.json` to format all js files in `TestingApp`

File: CONTRIBUTING.md (hunk 2 above)
Comment (prateek3255) [reply CONTRIBUTING.md:14]: Done

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**